[2023-11-24 22:35:00,625] INFO _log() L224   * Running on all addresses (0.0.0.0)
   WARNING: This is a development server. Do not use it in a production deployment.
 * Running on http://127.0.0.1:4001
 * Running on http://192.168.68.102:4001 (Press CTRL+C to quit) - call_trace=/Users/tairik/PycharmProjects/mvp-tairik/venv/lib/python3.9/site-packages/werkzeug/_internal.py L224 
[2023-11-24 22:35:02,494] INFO _log() L224  192.168.68.102 - - [24/Nov/2023 22:35:02] "[33mGET / HTTP/1.1[0m" 404 - - call_trace=/Users/tairik/PycharmProjects/mvp-tairik/venv/lib/python3.9/site-packages/werkzeug/_internal.py L224 
[2023-11-24 22:35:05,546] INFO _log() L224  192.168.68.102 - - [24/Nov/2023 22:35:05] "[33mGET /favicon.ico HTTP/1.1[0m" 404 - - call_trace=/Users/tairik/PycharmProjects/mvp-tairik/venv/lib/python3.9/site-packages/werkzeug/_internal.py L224 
[2023-11-24 22:35:13,023] INFO _log() L224  192.168.68.102 - - [24/Nov/2023 22:35:13] "[33mGET / HTTP/1.1[0m" 404 - - call_trace=/Users/tairik/PycharmProjects/mvp-tairik/venv/lib/python3.9/site-packages/werkzeug/_internal.py L224 
[2023-11-24 22:36:08,123] INFO _log() L224   * Running on all addresses (0.0.0.0)
   WARNING: This is a development server. Do not use it in a production deployment.
 * Running on http://127.0.0.1:4001
 * Running on http://192.168.68.102:4001 (Press CTRL+C to quit) - call_trace=/Users/tairik/PycharmProjects/mvp-tairik/venv/lib/python3.9/site-packages/werkzeug/_internal.py L224 
[2023-11-24 22:36:15,225] INFO _log() L224  192.168.68.102 - - [24/Nov/2023 22:36:15] "[32mGET / HTTP/1.1[0m" 302 - - call_trace=/Users/tairik/PycharmProjects/mvp-tairik/venv/lib/python3.9/site-packages/werkzeug/_internal.py L224 
[2023-11-24 22:36:15,348] INFO _log() L224  192.168.68.102 - - [24/Nov/2023 22:36:15] "GET /openapi/ HTTP/1.1" 200 - - call_trace=/Users/tairik/PycharmProjects/mvp-tairik/venv/lib/python3.9/site-packages/werkzeug/_internal.py L224 
[2023-11-24 22:36:15,491] INFO _log() L224  192.168.68.102 - - [24/Nov/2023 22:36:15] "[36mGET /openapi/static/images/swagger.svg HTTP/1.1[0m" 304 - - call_trace=/Users/tairik/PycharmProjects/mvp-tairik/venv/lib/python3.9/site-packages/werkzeug/_internal.py L224 
[2023-11-24 22:36:15,494] INFO _log() L224  192.168.68.102 - - [24/Nov/2023 22:36:15] "[36mGET /openapi/static/images/redoc.svg HTTP/1.1[0m" 304 - - call_trace=/Users/tairik/PycharmProjects/mvp-tairik/venv/lib/python3.9/site-packages/werkzeug/_internal.py L224 
[2023-11-24 22:36:16,004] INFO _log() L224  192.168.68.102 - - [24/Nov/2023 22:36:16] "[36mGET /openapi/static/images/rapidoc.svg HTTP/1.1[0m" 304 - - call_trace=/Users/tairik/PycharmProjects/mvp-tairik/venv/lib/python3.9/site-packages/werkzeug/_internal.py L224 
[2023-11-24 22:36:17,294] INFO _log() L224  192.168.68.102 - - [24/Nov/2023 22:36:17] "GET /openapi/swagger HTTP/1.1" 200 - - call_trace=/Users/tairik/PycharmProjects/mvp-tairik/venv/lib/python3.9/site-packages/werkzeug/_internal.py L224 
[2023-11-24 22:36:17,351] INFO _log() L224  192.168.68.102 - - [24/Nov/2023 22:36:17] "[36mGET /openapi/static/css/swagger-ui.css HTTP/1.1[0m" 304 - - call_trace=/Users/tairik/PycharmProjects/mvp-tairik/venv/lib/python3.9/site-packages/werkzeug/_internal.py L224 
[2023-11-24 22:36:17,356] INFO _log() L224  192.168.68.102 - - [24/Nov/2023 22:36:17] "[36mGET /openapi/static/js/swagger-ui-bundle.js HTTP/1.1[0m" 304 - - call_trace=/Users/tairik/PycharmProjects/mvp-tairik/venv/lib/python3.9/site-packages/werkzeug/_internal.py L224 
[2023-11-24 22:36:17,378] INFO _log() L224  192.168.68.102 - - [24/Nov/2023 22:36:17] "[36mGET /openapi/static/js/swagger-ui-standalone-preset.js HTTP/1.1[0m" 304 - - call_trace=/Users/tairik/PycharmProjects/mvp-tairik/venv/lib/python3.9/site-packages/werkzeug/_internal.py L224 
[2023-11-24 22:36:20,043] INFO _log() L224  192.168.68.102 - - [24/Nov/2023 22:36:20] "GET /openapi/openapi.json HTTP/1.1" 200 - - call_trace=/Users/tairik/PycharmProjects/mvp-tairik/venv/lib/python3.9/site-packages/werkzeug/_internal.py L224 
[2023-11-24 22:36:27,209] INFO _log() L224  192.168.68.102 - - [24/Nov/2023 22:36:27] "GET /raa HTTP/1.1" 200 - - call_trace=/Users/tairik/PycharmProjects/mvp-tairik/venv/lib/python3.9/site-packages/werkzeug/_internal.py L224 
[2023-11-24 22:43:21,603] INFO _log() L224   * Running on all addresses (0.0.0.0)
   WARNING: This is a development server. Do not use it in a production deployment.
 * Running on http://127.0.0.1:4001
 * Running on http://192.168.68.102:4001 (Press CTRL+C to quit) - call_trace=/Users/tairik/PycharmProjects/mvp-tairik/venv/lib/python3.9/site-packages/werkzeug/_internal.py L224 
[2023-11-24 22:43:24,487] ERROR log_exception() L1449 Exception on /raa [GET] - call_trace=/Users/tairik/PycharmProjects/mvp-tairik/venv/lib/python3.9/site-packages/flask/app.py L1449
Traceback (most recent call last):
  File "/Users/tairik/PycharmProjects/mvp-tairik/venv/lib/python3.9/site-packages/flask/app.py", line 2073, in wsgi_app
    response = self.full_dispatch_request()
  File "/Users/tairik/PycharmProjects/mvp-tairik/venv/lib/python3.9/site-packages/flask/app.py", line 1519, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/Users/tairik/PycharmProjects/mvp-tairik/venv/lib/python3.9/site-packages/flask_cors/extension.py", line 165, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
  File "/Users/tairik/PycharmProjects/mvp-tairik/venv/lib/python3.9/site-packages/flask/app.py", line 1517, in full_dispatch_request
    rv = self.dispatch_request()
  File "/Users/tairik/PycharmProjects/mvp-tairik/venv/lib/python3.9/site-packages/flask/app.py", line 1503, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**req.view_args)
  File "/Users/tairik/PycharmProjects/mvp-tairik/venv/lib/python3.9/site-packages/flask_openapi3/scaffold.py", line 109, in wrapper
    resp = _do_wrapper(
  File "/Users/tairik/PycharmProjects/mvp-tairik/venv/lib/python3.9/site-packages/flask_openapi3/do_wrapper.py", line 135, in _do_wrapper
    response = func(**request_kwargs)
  File "/Users/tairik/PycharmProjects/mvp-tairik/rest_api_pets/app.py", line 76, in get_raa
    return apresenta_raa(prediction), 200
NameError: name 'apresenta_raa' is not defined
[2023-11-24 22:43:24,517] INFO _log() L224  192.168.68.102 - - [24/Nov/2023 22:43:24] "[35m[1mGET /raa HTTP/1.1[0m" 500 - - call_trace=/Users/tairik/PycharmProjects/mvp-tairik/venv/lib/python3.9/site-packages/werkzeug/_internal.py L224 
[2023-11-24 22:43:42,855] ERROR log_exception() L1449 Exception on /raa [GET] - call_trace=/Users/tairik/PycharmProjects/mvp-tairik/venv/lib/python3.9/site-packages/flask/app.py L1449
Traceback (most recent call last):
  File "/Users/tairik/PycharmProjects/mvp-tairik/venv/lib/python3.9/site-packages/flask/app.py", line 2073, in wsgi_app
    response = self.full_dispatch_request()
  File "/Users/tairik/PycharmProjects/mvp-tairik/venv/lib/python3.9/site-packages/flask/app.py", line 1519, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/Users/tairik/PycharmProjects/mvp-tairik/venv/lib/python3.9/site-packages/flask_cors/extension.py", line 165, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
  File "/Users/tairik/PycharmProjects/mvp-tairik/venv/lib/python3.9/site-packages/flask/app.py", line 1517, in full_dispatch_request
    rv = self.dispatch_request()
  File "/Users/tairik/PycharmProjects/mvp-tairik/venv/lib/python3.9/site-packages/flask/app.py", line 1503, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**req.view_args)
  File "/Users/tairik/PycharmProjects/mvp-tairik/venv/lib/python3.9/site-packages/flask_openapi3/scaffold.py", line 109, in wrapper
    resp = _do_wrapper(
  File "/Users/tairik/PycharmProjects/mvp-tairik/venv/lib/python3.9/site-packages/flask_openapi3/do_wrapper.py", line 135, in _do_wrapper
    response = func(**request_kwargs)
  File "/Users/tairik/PycharmProjects/mvp-tairik/rest_api_pets/app.py", line 76, in get_raa
    return apresenta_raa(prediction), 200
NameError: name 'apresenta_raa' is not defined
[2023-11-24 22:43:42,858] INFO _log() L224  192.168.68.102 - - [24/Nov/2023 22:43:42] "[35m[1mGET /raa HTTP/1.1[0m" 500 - - call_trace=/Users/tairik/PycharmProjects/mvp-tairik/venv/lib/python3.9/site-packages/werkzeug/_internal.py L224 
[2023-11-24 22:43:50,818] INFO _log() L224   * Running on all addresses (0.0.0.0)
   WARNING: This is a development server. Do not use it in a production deployment.
 * Running on http://127.0.0.1:4001
 * Running on http://192.168.68.102:4001 (Press CTRL+C to quit) - call_trace=/Users/tairik/PycharmProjects/mvp-tairik/venv/lib/python3.9/site-packages/werkzeug/_internal.py L224 
